src/main/java/pl/jakubkozlowski/learning/firststeps/converter/ModelMapperConverter.java

```diff
@@ -6,13 +6,24 @@
 
 import java.util.Collections;
 import java.util.List;
+import java.util.function.Function;
 import java.util.stream.Collectors;
 
 public interface ModelMapperConverter<DTO, Entity> {
 
-
     default DTO convertEntity(Entity entity) {
-        return entity == null ? null : getModelMapper().map(entity, getDTOClass());
+        if (entity == null) {
+            return null;
+        }
+        return getModelMapper().map(entity, getDTOClass());
+    }
+
+    default DTO convertEntity(Entity entity, List<Function<DTO, DTO>> additionalConverters) {
+        if (entity == null) {
```

Comment:
it is already in used method of convertEntity. Use the fact that you only need to check if list of converters is not null or empty because if it is, you simply return your dto, null check is in line 15. already